deployment/utils.go

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package deployment
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/mattermost/mattermost-load-test-ng/deployment/terraform/ssh"
+)
+
+type Cmd struct {
+	Msg     string
+	Value   string
+	Clients []*ssh.Client
+}
+
+type DBSettings struct {
+	UserName string
+	Password string
+	DBName   string
+	Host     string
+	Engine   string
+}
+
+// ProvisionURL takes a URL pointing to a file to be provisioned.
```

Comment:
I know this comment is copied over from the other location, but could we specify here that we can use both http(s):// and file:// as protocols here in the docs?

Reply:
done